react/typings/countryRulesTypes.d.ts

```diff
@@ -5,3 +5,38 @@ export interface LineFragment {
   delimiter?: string
   delimiterAfter?: string
 }
+
+export interface Field {
+  label: string
+  hidden?: boolean
+  maxLength?: number
+  size?: string
+  required?: boolean
+  autoComplete?: string
+  optionsCaption?: string
+  options?: ({ label: string; value: string })[]
```

Comment:
you can also declare this as `Array<{ label: string; value: string }>` if you prefer that

Reply:
It seems that this is prohibited by our linter: `Array type using 'Array<T>' is forbidden. Use 'T[]' instead` 😢